test/Farmer/addProduct.test.js

```diff
@@ -0,0 +1,53 @@
+import { expect } from "chai";
```

Comment:
Codacy has a fix for the issue: [Strings must use singlequote.](https://app.codacy.com/gh/JoeMindo/USSDTest/pullRequest?prid=8695127)
```suggestion
import { expect } from 'chai';
```